client/sites-dashboard/components/no-sites-message.tsx

```diff
@@ -8,15 +8,32 @@ const NoSitesLayout = styled( EmptyContent )`
 	flex-direction: column;
 	justify-content: center;
 	align-items: center;
+
+	${ ( { illustration } ) =>
+		! illustration && {
+			marginTop: '10%',
+		} }
+
+	.empty-content__line {
+		margin-bottom: 20px;
+	}
+
+	.empty-content__illustration {
```

Comment:
We used styled components in no sites component, but we need to style child component UI here. Is there a better way to style those in this context?

Reply:
I'm not certain what's the best practice here but it seems good to me to override the underlying style classes of the `EmptyContent` component as you are doing here. It prevents using additional elements to style the component how we want.

Reply:
Normally, I don't like nesting selectors in the components, however this is styling a nested element that is used inside itself. So I don't see the issue because we're customizing a DOM element that is not exposed. I think it's fine.